src/resnet.jl

```diff
@@ -50,17 +50,38 @@ function trained_resnet50_layers()
   weight = Metalhead.weights("resnet.bson")
   weights = Dict{Any ,Any}()
   for ele in keys(weight)
-    weights[string(ele)] = convert(Array{Float64, N} where N, weight[ele])
+    weights[string(ele)] = weight[ele]
   end
   ls = load_resnet(resnet_configs["resnet50"]...)
-  ls[1].weight.data .= flipkernel(weights["gpu_0/conv1_w_0"])
+  ls[1][1].weight.data .= flipkernel(weights["gpu_0/conv1_w_0"])
+  ls[1][2].σ² .= weights["gpu_0/res_conv1_bn_riv_0"]
+  ls[1][2].layers[2].μ .= weights["gpu_0/res_conv1_bn_rm_0"]
```

Comment:
@avik-pal I didn't understand this line. If ls[1][2] isa BatchNorm layer, then how/why do you put `.layers[2]` here ?

Reply:
That's a copy-paste error.